docs/dev.md

```diff
@@ -127,6 +128,49 @@ reader, ec, err := config.SonobuoyClient.RetrieveResults(&client.RetrieveConfig{
 })
 ```
 
+#### Sonobuoy image mirroring <a name="dev-image-mirror-sonobuoy"></a>
+
+The Sonobuoy images for Aggregator server and worker is mirrored to quay.io to prevent
+issues with docker hub network throtling.
```

Comment:
```suggestion
The Sonobuoy images for Aggregator server and worker are mirrored to quay.io to prevent
issues with docker hub network throttling.
```